src/Components/SubmitButton.js

```diff
@@ -10,23 +10,42 @@ const SubmitButton = (props) => {
     e.preventDefault();
     let type = props.type;
     let path = "";
+    let redirectPath = "";
     if(type == "landingPage") {
-      path = `/users/exists?email=${props.email}`;
+      axios.get(`http://localhost:8080/users/exists?email=${props.email}`)
+      .then((res) => {
+          if (res.status == 200) {
+              console.log(res);
+              if(!res.data.exists) {
+                redirectPath = "/signup";
+              }
+              else { 
+                redirectPath = "/login"
+              }
+              history.push({pathname: redirectPath, email: props.email})
+          }
+      })
     }
-    axios.get(`http://localhost:8080${path}`)
-    .then((res) => {
-        if (res.status == 200) {
+    else if (type=="createEvent") {
+        axios.post('http://localhost:8080/events', {
+            eventName: props.eventName,
+            exDescription: props.exDescription,
+            inDescription: props.inDescription,
+            address: props.address,
+            city: props.city,
+            link: props.link
+            // TODO: this should have more data but not sure what to do yet for elements that exist only in mdb pro (select/datepicker/timepicker)
```

Comment:
@zhengwin can we pair up with Darren and figure this out in an hour or so. Seems like a blocker.